.github/workflows/release.yml

```diff
@@ -0,0 +1,63 @@
+# Trigger this workflow on release
+on:
+  release:
+    types: [published]
+
+# Make sure the GITHUB_TOKEN has permission to upload to our releases
+permissions:
+  contents: write
+
+jobs:
+  linux:
+    name: "Build Linux standalone version"
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout the repository"
+        uses: actions/checkout@v4
+        with:
+          submodules: true
+
+      - name: "Set up Python"
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.11"
+
+      - name: "Build YOGA Linux Standelone Version"
+        run: |
+          linux-bin/build.sh
+
+      - name: Archive Linux Build
+        run: |
+          zip yoga-${{ github.event.release.tag_name }}_linux_standalone_x86-64.zip -r yoga-bin.dist
```

Comment:
It is better to generate a tarball for Unix systems (`.tar.gz`). They a more standard on those platforms.